ssl/quic/quic_ackm.c

```diff
@@ -1028,6 +1044,10 @@ OSSL_ACKM *ossl_ackm_new(OSSL_TIME (*now)(void *arg),
     ackm->statm     = statm;
     ackm->cc_method = cc_method;
     ackm->cc_data   = cc_data;
+
+    ackm->rx_max_ack_delay = ossl_ms2time(QUIC_DEFAULT_MAX_ACK_DELAY);
+    ackm->tx_max_ack_delay = DEFAULT_TX_MAX_ACK_DELAY;
```

Comment:
I'm left wondering why you introduced a macro for DEFAULT_TX_MAX_ACK_DELAY, but there isn't one for the RX side. The TX macro expands to exactly the same thing as the RX side.

Reply:
Ah, right &mdash; so the reason for this is that the RX max ACK delay is a QUIC protocol value communicated via a transport parameter and explicitly denominated in milliseconds. The default value for this is also set by the RFC, so it's essentially a protocol constant and by definition in milliseconds.

By comparison, the TX maximum ACK delay is not signalled in the protocol and is up to us to choose arbitrarily. So we don't have to choose it with millisecond precision.

Subtleties...

Reply:
We have the defaults set from the QUIC_CHANNEL. Do we need to set them here as well?

Reply:
Maybe not... though I'd prefer to leave things initialised to a sensible default within the ACKM module. A lot of test units make their own instance of the ACKM, so it would mean adding calls to all those to set these values up right.